guardian/mixins.py

```diff
@@ -160,6 +167,16 @@ def get_required_permissions(self, request=None):
                                        % self.permission_required)
         return perms
 
+    def get_permission_denied_message(self):
+        """
+        Returns the message to be passed to the ``PermissionDenied`` exception. By default,
+        it returns the value from the ``permission_denied_message`` attribute.
+        """
+        message = ""
+        if isinstance(self.permission_denied_message, str):
+            message = self.permission_denied_message
+        return message
```

Comment:
And I think this function is a little complex.
In `django.contrib.auth.mixins.PermissionRequiredMixin`, we can find that it just return the `self.permission_denied_message`.
```
# in django/contrib/auth/mixins/PermissionRequiredMixin
    def get_permission_denied_message(self):
        """
        Override this method to override the permission_denied_message attribute.
        """
        return self.permission_denied_message
```